backend/database/migrations/2023_01_26_040851_create_teams_table.php

```diff
@@ -17,7 +17,7 @@ public function up()
             $table->id();
             $table->string('name');
             $table->string('description');
-            $table->string('dashboard_content');
+            $table->longText('dashboard_content');
```

Comment:
longtext is very big not optimized. You dont expect a longtext for a dashboard content, 
```suggestion
            $table->text('dashboard_content');
```

Reply:
I tried adding an image in the content sir and it was converted into a very long string just for 1 image.

I'll try to use `text` instead of `longText` and update you asap sir 🙇🏽‍♂️

Reply:
I tried using `text` sir and it produces an error when I'm trying to add an image.

`SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'dashboard_content' at row 1`

Reply:
I think `mediumText` will be the better option sir 🤔